sample-config/drone.toml

```diff
@@ -58,7 +58,7 @@ ip = "123.123.123.123"
 # Optional Docker settings for the agent.
 [agent.docker]
 # The runtime to use (defaults to "runc")
-runtime = "runsc"
+runtime = "runc"
```

Comment:
does this need to be in the sample config?

Reply:
Functionally, it doesn't need to be because it's the default, but I'd like for this to double as documentation of the toml format until we have better docs.